libs/auth/src/common/services/login-strategies/login-strategy.service.ts

```diff
@@ -47,6 +46,7 @@ import {
   KdfConfigService,
 } from "@bitwarden/key-management";
 
+import { VaultTimeoutSettingsService } from "../../../../../key-management/src/vault-timeout/abstractions/vault-timeout-settings.service";
```

Comment:
Should probably be something like:

```suggestion
import { VaultTimeoutSettingsService } from "@bitwarden/key-management"
```